docs/setup/settings.asciidoc

```diff
@@ -296,14 +308,137 @@ This value must be a whole number greater than zero. *Default: `"1000"`*
 Maximum number of documents loaded by each shard to generate autocomplete
 suggestions. This value must be a whole number greater than zero.
 *Default: `"100000"`*
+
+[WARNING]
+============================================================================
+While {kib} supports both deprecated and current logging systems being enabled at the same time, mixing the configurations is not recommended. To avoid unforeseen consequences, rather switch over to the current configuration completely.
```

Comment:
super nit
```suggestion
While {kib} supports both deprecated and current logging systems being enabled at the same time, mixing the configurations is not recommended. To avoid unforeseen consequences, we recommend switching over to the current configuration completely.
```